### PR TITLE
Some me model cleanup

### PR DESCRIPTION
**Pull request by estrattonbailey**

### CODE REVIEW

File: src/view/com/composer/Prompt.tsx [hunk 1]
```diff
@@ -3,13 +3,15 @@ import {StyleSheet, TouchableOpacity} from 'react-native'
...
 import {useWebMediaQueries} from 'lib/hooks/useWebMediaQueries'
 import {Trans, msg} from '@lingui/macro'
 import {useLingui} from '@lingui/react'
+import {useSession} from '#/state/session'
+import {useProfileQuery} from '#/state/queries/profile'
 
 export function ComposePrompt({onPressCompose}: {onPressCompose: () => void}) {
-  const store = useStores()
+  const {currentAccount} = useSession()
+  const {data: profile} = useProfileQuery({did: currentAccount!.did})
```
Comment (pfrazee) [inline src/view/com/composer/Prompt.tsx:14]: This is actually a place where we want to handle the possibility of no session

File: src/view/shell/bottom-bar/BottomBar.tsx [hunk 2]
```diff
@@ -47,6 +49,7 @@ export const BottomBar = observer(function BottomBarImpl({
     useNavigationTabState()
   const numUnreadNotifications = useUnreadNotifications()
   const {footerMinimalShellTransform} = useMinimalShellMode()
+  const {data: profile} = useProfileQuery({did: currentAccount!.did})
```
Comment (pfrazee) [inline src/view/shell/bottom-bar/BottomBar.tsx:52]: Another where we need to handle it

**Review by pfrazee — CHANGES_REQUESTED** [review]
Great - I think we just need to not use `currentAccount!` because that'll reference error. If the profile query doesn't support `enabled: false` on no `did` yet, you can add that to avoid the conditional hook

File: src/view/com/composer/Prompt.tsx (hunk 1 above)
Comment (estrattonbailey) [reply src/view/com/composer/Prompt.tsx:14]: Yep yep, the `useProfileQuery` has the `enabled` check, and `UserAvatar` falls back to `DefaultAvatar` in the case of undefined. That work or were you thinking of something else?

File: src/view/shell/bottom-bar/BottomBar.tsx (hunk 2 above)
Comment (estrattonbailey) [reply src/view/shell/bottom-bar/BottomBar.tsx:51]: Updated this to match the above, will fall back to default avi

File: src/view/com/composer/Prompt.tsx (hunk 1 above)
Comment (estrattonbailey) [reply src/view/com/composer/Prompt.tsx:14]: Tbh I'd rather not rely on those enable checks much if we can help it, but we'll need to for the time being

Comment (pfrazee) [reply src/view/com/composer/Prompt.tsx:14]: Wont `currentAccount!.did` reference error if `currentAccount` is undefined?

**Review by pfrazee — APPROVED** [review]

**Pull request merged**